manage.py

```diff
@@ -23,13 +23,17 @@
 
 terraform <cmd> -var-file=../terraform.tfvars -var-file=../variables.json
 """
+# pylint: disable=too-many-lines
```

Comment:
Haha I know - but there is _literally_ nothing in here besides argparse stuff right now. I'm open to suggestions for how to split that, but splitting up the argparse logic would be a PITA and cause weird fragmentation.

Reply:
Yeah, that's fine. So instead of disabling the pylint check, change the `.pylintrc` to allow longer files. Either way is fine tho

Reply:
@austinbyers I didn't add it because I don't really want to promote longer files project wide, and would rather do case-by-case exceptions like this. Is there a way to do that in the rc?